welcome/revhistory_full.adoc

```diff
@@ -5,8 +5,23 @@
 :icons:
 :experimental:
 
-The following sections aggregate the revision histories of each guide by publish
-date.
+The following sections aggregate the revision histories of each guide by publish date.
+
+== Thu Mar 17 2016
+.Architecture
+include::architecture/revhistory_architecture.adoc[tag=architecture_thu_mar_17_2016]
+
+.Release Notes
+include::release_notes/revhistory_release_notes.adoc[tag=release_notes_thu_mar_17_2016]
+
+.Dev Guide
+include::dev_guide/revhistory_dev_guide.adoc[tag=dev_guide_thu_mar_17_2016]
+
+.Install Config
+include::install_config/revhistory_install_config.adoc[tag=install_config_thu_mar_17_2016]
+
+.Admin Guide
+include::admin_guide/revhistory_admin_guide.adoc[tag=admin_guide_thu_mar_17_2016]
```

Comment:
@vikram-redhat "Dev", "Install Config", and "Admin" should be spelled out. We can touch it up in next week's update (cc @tnguyen-rh).